code/modules/mob/living/simple_animal/constructs/constructs.dm

```diff
@@ -233,6 +237,7 @@
 	resistance = 10
 	var/energy = 0
 	var/max_energy = 1000
+	can_climb = 1 //It flies, its magic
```

Comment:
I concur. Above all cult constructs being able to climb ladders is ridiculous. Most of them cannot fly, and none of them have proper hands. As a sub-antagonist there is _no_ problem in having them to rely on their master, especially when their masters can easily draw up teleportation runes.